src/vec.rs

```diff
@@ -606,6 +660,25 @@ where
             .expect("Replaced old value");
     }
 
+    /// Try to add the given element to the `identified_vec` unconditionally, either appending it to the `identified_vec``, or
```

Comment:
"Unconditionally" is not true right? There is a condition: must exist one with the ID

Reply:
Well spotted, fixed it.